### DISCUSSION TITLE
Error opening ExodusII mesh file: /home/ubuntu/cardinal/tutorials/lwr_solid/mesh_in.e

### DISCUSSION THREAD

**Discussion by vincemilo** [neams-th-coe/cardinal#520] [Q&A] [answered]
Tried to check the install outlined in https://cardinal.cels.anl.gov/with_conda.html by running:

`mpiexec -np 2 ../../cardinal-opt -i solid.i --n-threads=2`

And got:

```
*** Warning, This code is deprecated and will be removed in future versions:
Deprecated Object: MultiAppMeshFunctionTransfer
This object will be removed on Mon Jan  1 01:00:00 2024
Replace MultiAppMeshFunctionTransfer with MultiAppShapeEvaluationTransfer


Exodus Library Warning/Error: [ex__check_file_type]
	ERROR: Could not open file '/home/ubuntu/cardinal/tutorials/lwr_solid/mesh_in.e', error = No such file or directory.

Exodus Library Warning/Error: [ex_open_int]
	ERROR: failed to open /home/ubuntu/cardinal/tutorials/lwr_solid/mesh_in.e of type 0 for reading. Either the file does not exist,
	or there is a permission or file format issue.
Error opening ExodusII mesh file: /home/ubuntu/cardinal/tutorials/lwr_solid/mesh_in.e
[0] ../src/mesh/exodusII_io_helper.C, line 813, compiled Nov  8 2022 at 20:40:06
```

**Comment by aprilnovak** [comment, marked as answer]
Hi @vincemilo - that error is showing that the file `mesh_in.e` could not be found. In the tutorial, you're first supposed to generate the mesh yourself. Look in the vicinity of [this section of the tutorial](https://cardinal.cels.anl.gov/tutorials/pincell1.html#:~:text=meshgenerators%20are%20used%20to%20construct%20the%20solid%20mesh.) - you'll need to run

```
cardinal-opt -i mesh.i --mesh-only
```

**Reply by aprilnovak** [reply]
Ah, I see - the example I'm telling you to run requires extra steps that wouldn't be obvious if you're on the "getting started" page. I will change this to use a better example that doesn't require any other prep steps. 

In the meantime, you can check the install with this one:

```
cd cardinal/test/tests/neutronics/feedback/lattice
cardinal-opt -i openmc_master.i
```

**Reply by aprilnovak** [reply]
Fixed in #522. If you refresh the website page, you should see the new instructions: https://cardinal.cels.anl.gov/with_conda.html

Sorry about that oversight, and thank you for reporting it!

**Reply by vincemilo** [reply]
Thanks!
